packages/dashboard-frontend/src/store/FactoryResolver/index.ts

```diff
@@ -212,8 +212,7 @@ export const actionCreators: ActionCreators = {
             throw response.data;
           }
         }
-        const errorMessage =
-          'Failed to request factory resolver: ' + common.helpers.errors.getMessage(e);
```

Comment:
I was just wondering why did you change this error message?

Reply:
I removed this additional because the original error message starts from the same information.